src/saving/SaveHelper.cs

```diff
@@ -81,6 +104,9 @@ public static void Save(string name, MicrobeEditor state)
     /// <param name="state">Data to include in save</param>
     public static void QuickSave(MicrobeStage state)
     {
+        if (!AllowQuickSavingAndLoading)
```

Comment:
Maybe InternalSaveHelper should check this? As we'll end up with 16 quick save variants (at least, one per stage and editor, and with much more editors later in the game, we might end up with a ton of these checks being duplicated).

Reply:
Yeah that's preferrable, I actually didn't saw that there's a SaveType parameter that you can use to distinguish saves, that was why I done it like this.